examples/bench.py

```diff
@@ -0,0 +1,53 @@
+# This is the current base speed performance benchmarking script.
+# For actual timing runs, it may need to be modified to set the
+# numbers of CPUs, GPUs, batches, devices, etc.  Other Sycamore
+# code may also need to be changed to adjust actor pools.  The
+# script should be run similar to this:
+#
+# TIMETRACE=/tmp/tt poetry run python examples/bench.py
+
+import sys
+import pyarrow.fs
+
+# ruff: noqa: E402
+sys.path.append("../sycamore")
```

Comment:
Is this necessary?

Reply:
Seems like since the monorepo reorg, no.